website/src/script/popup.ts

```diff
@@ -32,7 +54,10 @@ export default function (
     }
   }
 
-  return html + `<div class="popup-streetname">${streetname}</div>`;
+  html += `<div class="popup-streetname">${streetname}`;
+  html += `<a target="_blank" href="https://edit.equalstreetnames.eu/?type=${featureType}&id=${featureId}&close=1" class="fas fa-edit popup-edit" title="Edit in OpenStreetMap"></a></div>`;
```

Comment:
```suggestion
  html += '<div class="popup-streetname">';
  html += streetname;
  html += `<a target="_blank" href="https://edit.equalstreetnames.eu/?type=${featureType}&id=${featureId}&close=1" class="fas fa-edit popup-edit" title="Edit in OpenStreetMap"></a>`;
  html += '</div>';
```

Reply:
Is https://edit.equalstreetnames.eu/ production ready ?
For now I would link to `https://www.openstreetmap.org/relation/12345` or `https://www.openstreetmap.org/way/12345`.

Reply:
@jbelien I believe it should be relatively stable now, but I'll let it link to iD for now.

Reply:
I would now link it to iD but rather to the element on https://osm.org/ because you need to be logged in to access iD editor.
That would also allow people to get all the data available in _OpenStreetMap_ for a specific relation/way.